frontend/src/components/supp-allowances/IndigenousProgrammingAllowance.vue

```diff
@@ -1,3 +1,158 @@
 <template>
-  <div>This is a placeholder for Indigenous Programming Allowance form</div>
+  <v-row no-gutters class="mr-2 my-2">
+    <v-col cols="12">
+      <ul>
+        <li><AppLabel>Purpose of the Fund:</AppLabel></li>
+      </ul>
+      The Indigenous Programming Allowance is intended to provide additional funds to child care providers to support the participant’s delivery of Indigenous curriculum in collaboration with
+      Indigenous Peoples in their community. To reduce barriers to accessing these funds, all expenses are considered eligible, except for staffing enhancements for the purpose of reducing ratio. 
+      Providers may reach out to BC Aboriginal Child Care Society (BCACCS), Métis Nation of BC (MNBC), and/or the Indigenous Perspectives Society (IPS) for resources that support child care providers
+      to enhance the their programs.
+    </v-col>
+  </v-row>
+  <br />
+  <v-row no-gutters class="mr-2 my-2">
+    <v-col cols="12">
+      <ul>
+        <li>
+          <AppLabel>Funding Amount:</AppLabel>
+          Child care facilities may access the Indigenous Programming Allowance annually, as follows:
+        </li>
+      </ul>
+      <v-row no-gutters>
+        <v-col cols="12" class="ml-10 my-2">
+          <ul>
+            <li>$5,000 for Facilities with an Operational Capacity of more than 20 children</li>
+            <li>$3,500 for Facilities with an Operational Capacity of between 10-20 children</li>
+            <li>$2,000 for Facilities with an Operational Capacity of 9 or less children</li>
+          </ul>
+        </v-col>
+      </v-row>
+    </v-col>
+  </v-row>
+  <v-divider class="mt-2"></v-divider>
+
+  <v-row no-gutters class="mr-2 my-2">
+    <v-col cols="12">
+      Ineligible expences include: Staffing enhencements for the purposes of reducing ratio.
+      <br />
+      <br />
+      Please describe how you intend to use this funding
+      <strong>(Eligible Expences may include, but are not limited to):</strong>
+    </v-col>
+  </v-row>
+  <v-row no-gutters class="mr-2 my-0">
+    <v-checkbox
```

Comment:
I would suggest to use a v-for loop here to avoid duplicated code of "v-checkbox". We can create a CHECKBOX_LABELS constant in created() and just loop through though label and value of checkbox can be its index